templates/host-master.xml.erb

```diff
@@ -0,0 +1,191 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<host xmlns="urn:jboss:domain:10.0" name="master">
```

Comment:
I checked one of our older Keycloak 4.8.3 instances and they have this:

```<host xmlns="urn:jboss:domain:8.0" name="master">```

I wonder what guarantees we have that these relatively static XML files will work across Keycloak versions? I do not like the idea of having to use Augeas to modify these files, but having a diff between the vanilla version on this Puppet-managed-file would be nice to understand what we are changing exactly.

Reply:
I see some variable interpolations in this file and host-slave.xml.erb. Are those the only changes made to the files provided by the Keycloak project?